azext_edge/edge/params.py

```diff
@@ -576,6 +576,13 @@ def load_iotops_arguments(self, _):
             options_list=["--endpoint"],
             help="Asset endpoint name.",
         )
+        context.argument(
+            "custom_attributes",
+            options_list=["--custom-attribute", "--ca"],
```

Comment:
Something other than --ca for short alias? We have a lot of `--ca-*` parameters for certificate matters.

Reply:
my other idea was to use `--attribute` and `--attr` or `-a`

Reply:
Of those I vote for --attr